operator/v1alpha1/0000_10_config-operator_01_olm.crd.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1504
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/feature-set: TechPreviewNoUpgrade
```

Comment:
Do you expect anyone to every run a CustomNoUpgrade cluster and possibly want to use this feature? If you do (can be useful for isolated testing), we will need a CustomNoUpgrade version of the CRD as well, else CVO won't install the CRD at all

Reply:
Ever? Probably. 
In 4.14? It hasn't been discussed, and isn't a priority.

If it is literally just a matter of copying this CRD and using `CustomNoUpgrade` here instead, I have no problem doing that now.

Reply:
Added a `CustomNoUpgrade` CRD.